app/actions/TrezorActions.js

```diff
@@ -49,6 +49,11 @@ const OP_SSGEN_STR = "bb";
 const OP_SSRTX_STR = "bc";
 const STAKE_REVOCATION = "SSRTX";
 const STAKE_GENERATION = "SSGen";
+// VOTING_XPRIV_KEY and VOTING_XPRIV_VALUE must never change in order to
+// have deterministic voting keys.
+const VOTING_XPRIV_KEY = "Create decred voting xpriv seed bytes?";
```

Comment:
We may be able to get around this by using an IV.

Reply:
An IV has nothing to do with the immutability of the key, it's only used as part of the input to the encryption algorithm.

Reply:
An IV has nothing to do with the immutability of the key, it's only used as part of the input to the encryption algorithm.

Reply:
I see. The key must not change then.